client/src/store/actions/messageActions.js

```diff
@@ -0,0 +1,15 @@
+import { createAction } from 'redux-actions';
+import { webSocketActions } from './';
+
+const addMessages = createAction('ADD_MESSAGE');
+
+const sendMessage = (messageText) => (dispatch) => {
+    dispatch(webSocketActions.sendWebSocket(messageText));
+    const message = { data: messageText, from: { id: 'me', name: 'me' } }
```

Comment:
why is this hard coded ?

Reply:
again I can put it in a const but I don't think this should be configurable.
in the near future i want to replace it with the user name the user inputed